openapi.yaml

```diff
@@ -198,6 +198,21 @@ paths:
           $ref: '#/components/responses/BadRequest'
         "404":
           $ref: '#/components/responses/NotFound'
+    delete:
+      summary: Delete ticket
+      description: Delete ticket and refund. Allows for ticket cancellation.
+      operationId: deleteTicket
```

Comment:
It's a good idea to allow a refund. But delete isn't quite the right approach for that. Delete is similar to making it so that the ticket never existed. Instead, the ticket is really being revoked and invalidated and the payment is being refunded. We probably need to keep a record of these events.

Reply:
Ah good point! I'll close this.